layouts/partials/openapi/render-object-table.html

```diff
@@ -97,6 +107,27 @@
 
     {{ end }}
 
+ {{/* If additionalProperties is true, do nothing.
+      If additionalProperties is nil, treat is as the default (true) and do nothing.
+      If additionalProperties is false, raise a build-time error:
+      "Everything is extensible in Matrix unless otherwise noted".
+      Otherwise, it should be an OpenAPI schema object. Include the description
+      of that schema object, if it exists. */}}
+ {{ if (and (ne $additionalProperties nil) (ne $additionalProperties true)) }}
+  {{ if (and (reflect.IsMap $additionalProperties)) -}}
```

Comment:
can we restructure these `if` statements to reduce the nesting?

```
 {{ if (or (eq $additionalProperties nil) (eq $additionalProperties true)) }}
     {{/* do nothing */}}
  {{ else if (reflect.IsMap $additionalProperties) -}}
     ...
  {{ else if (not $additionalProperties) -}}
    ... 
  {{ end }}
```

or something.